src/elementary/IsotopeID.hpp

```diff
@@ -75,6 +79,14 @@ namespace elementary {
       return this->element_.number() * 1000 + this->mass_;
     }
 
+    /**
+     *  @brief Return the hash value associated to the isotope
+     */
+    auto hash() const noexcept {
+
+      return this->za();

```

Comment:
Is your `za` robust enough to handle metastables as different from ground state?

Reply:
This is the IsotopeID, it is used to identify only the general case for Z and A. Metastable states are handled in the NuclideID and NucleusID.

Reply:
FYI: PoPs uses such a concept so I created it as well.